app/src/test/java/org/mozilla/fenix/trackingprotection/TrackingProtectionStoreTest.kt

```diff
@@ -133,14 +135,16 @@ class TrackingProtectionStoreTest {
         url = "www.mozilla.org",
         isTrackingProtectionEnabled = true,
         listTrackers = listOf(),
-        mode = TrackingProtectionState.Mode.Normal
+        mode = TrackingProtectionState.Mode.Normal,
+        lastAccessedCategory = ""
```

Comment:
@ekager 
Sorry to bother you again with this. While modifying the code to use the `TrackingProtectionCategory` enum I realised why I chose to use strings instead. It's because I needed an empty value and didn't want to use nullable or add a NONE type to the enum. I hope I'm not overlooking anything.